hadoop-hdfs-project/hadoop-hdfs-client/src/test/java/org/apache/hadoop/hdfs/server/namenode/ha/TestRequestHedgingProxyProvider.java

```diff
@@ -98,9 +100,10 @@ public long[] answer(InvocationOnMock invocation) throws Throwable {
     final ClientProtocol badMock = Mockito.mock(ClientProtocol.class);
     Mockito.when(badMock.getStats()).thenThrow(new IOException("Bad mock !!"));
 
-    RequestHedgingProxyProvider<ClientProtocol> provider =
-        new RequestHedgingProxyProvider<>(conf, nnUri, ClientProtocol.class,
-            createFactory(badMock, goodMock));
+    RequestHedgingProxyProvider<ClientProtocol> provider = new RequestHedgingProxyProvider<>(
```

Comment:
Although it looks like max 80 chars a line is not strictly enforced in test package by checkstyle check, it would be nice to still follow this rule, so that the coding style is consistent.

Reply:
Sure will follow it consistently .